crates/oxc_linter/src/snapshots/no_side_effects_in_initialization.snap

```diff
@@ -541,18 +541,6 @@ source: crates/oxc_linter/src/tester.rs
    ·            ─
    ╰────
 
-  ⚠ eslint-plugin-tree-shaking(no-side-effects-in-initialization): Cannot determine side-effects of calling function parameter
```

Comment:
These are deleted snap because of repeat, previous version didn't remember same function call.